hypha/static_src/src/sass/apply/components/_heading.scss

```diff
@@ -6,16 +6,12 @@
     &--meta {
         display: flex;
         flex-wrap: wrap;
-        margin: 0;
-
-        @include media-query(tablet-portrait) {
-            margin-bottom: 1rem;
-        }
+        margin-top: .25rem;
 
         span {
             &::after {
-                padding: 0 15px;
-                content: '|';
+                padding: 0 .25rem;
+                content: '•';
```

Comment:
I personally prefer the pipe over the bullet but let's see what staff thinks. Most likely they do not even notice the change :-).

Reply:
Sure.